templates/openwisp2/settings.py

```diff
@@ -479,7 +479,7 @@
 }
 
 INSTALLED_APPS.append('djcelery_email')
-EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
+EMAIL_BACKEND = '{{ openwisp2_email_backend }}'
 {% endif %}
```

Comment:
```suggestion
{% endif %}

EMAIL_BACKEND = '{{ openwisp2_email_backend }}'
```

The email backend should not depend on availability of the monitoring module